.github/workflows/build.yml

```diff
@@ -184,3 +184,21 @@ jobs:
         asset_path: ./selene-light-macos.zip
         asset_name: selene-light-${{ env.VERSION }}-macos.zip
         asset_content_type: application/zip
+    - uses: actions/checkout@v1
+    - name: Log in to Docker Hub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+    - name: Extract metadata (tags, labels) for Docker
+      id: meta
+      uses: docker/metadata-action@v3
+      with:
+        images: henriquehbr/selene

```

Comment:
I see. I figured tag meant something like it does on GitHub labels. I'll try and get you one.

Reply:
Can you use `boyned/selene`? I just made a new repository, not sure if I need to do anything else.

Reply:
Just updated the Docker image namespace, for now, I think that's it, let's see if everything goes fine in the pipelines and the image is properly published, otherwise we look into a fix for whatever went wrong